geode-core/src/test/java/org/apache/geode/internal/DataSerializableJUnitTest.java

```diff
@@ -1166,13 +1166,13 @@ public void testVersionedDataSerializable() throws Exception {
     VersionedDataSerializableImpl ds = new VersionedDataSerializableImpl(getRandom());
 
     VersionedDataOutputStream v =
-        new VersionedDataOutputStream(this.baos, KnownVersion.GFE_70);
+        new VersionedDataOutputStream(this.baos, KnownVersion.GEODE_1_11_0);
```

Comment:
Is there a specific reason for this version to be used here? Similarly for the use of `GEODE_1_12_0` below.